tools/android/packaging/xbmc/searchable.xml.in

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<searchable xmlns:android="http://schemas.android.com/apk/res/android"
+    android:label="@string/app_name"
+    android:hint="@string/search_hint" 
+
+    android:searchSuggestAuthority="org.xbmc.kodi.media"
+    android:searchSuggestPath="suggestions"
+    
+    android:searchSuggestIntentAction="android.intent.action.GET_CONTENT"
+    android:searchSuggestIntentData="content://org.xbmc.kodi.media/intent"
```

Comment:
not sure if this file is generated by the build sys, but @APP_NAME_LC@ applies here as well